include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -88,7 +88,7 @@ struct layout_stride {
     : private detail::__no_unique_address_emulation<
         detail::__compressed_pair<
           Extents,
-          std::array<typename Extents::index_type, Extents::rank()>
+          detail::possibly_empty_array<typename Extents::index_type, Extents::rank()>
```

Comment:
see below: the base_t typedef was already using possibly_empty_array. This needed to change, because std::array is not actually empty for rank == 0